iris_grib/tests/unit/grib1_load_rules/test_grib1_convert.py

```diff
@@ -116,20 +116,17 @@ def test_grib1_hybrid_height(self):
         self.assertEqual(sigma, {'long_name': 'sigma'})
         self.assertEqual(ref, Reference(name='surface_pressure'))
 
-        ml_ref = iris.coords.CoordDefn('model_level_number', None, None,
-                                       cf_units.Unit('1'),
-                                       {'positive': 'up'}, None, False)
-        lp_ref = iris.coords.CoordDefn(None, 'level_pressure', None,
-                                       cf_units.Unit('Pa'),
-                                       {}, None, False)
-        s_ref = iris.coords.CoordDefn(None, 'sigma', None,
-                                      cf_units.Unit('1'),
-                                      {}, None, False)
-
-        aux_coord_defns = [coord._as_defn() for coord, dim in results[8]]
-        self.assertIn(ml_ref, aux_coord_defns)
-        self.assertIn(lp_ref, aux_coord_defns)
-        self.assertIn(s_ref, aux_coord_defns)
+        coords_and_dims = results[8]
+        coord, = [co for co, _ in coords_and_dims
+                  if co.name() == 'model_level_number']
+        self.assertEqual(coord.units, '1')
+        self.assertEqual(coord.attributes['positive'], 'up')
```

Comment:
Should you also assert empty `attributes` for the other two `coord`s?

Reply:
Well, I thought it was OK to just check the things which the convert routine produces.
My reading of the previous code is that it **_has_** to provide all args to create a CoordDefn, but it's not really  important to check every aspect.